azure-ai-search/index_search/__init__.py

```diff
@@ -14,19 +15,53 @@ class EmptyQueryError(Exception):
 
 @dataclass
 class AzureIndexSearchConfig:
-    endpoint: str
-    api_key: str
-    index_name: str
+    """
+    Configuration class for Azure Index Search.
+
+    This class holds the necessary configuration details required for setting up
```

Comment:
this paragraph doesn't seem to wrap on 80

Reply:
Ruff allows up to 88 chars per line inside the code. Could be configured to 80. The description is also too verbose imo so I shortened it.